### DISCUSSION TITLE
Flash Sonoof HAA RavenCore

### DISCUSSION THREAD

**Discussion by msmicka** [RavenSystem/esp-homekit-devices#1575] [General]
Flash Sonoff MINI R2 

Today I discovered a strange problem. With FLASH Sonoff mini R2. Everything goes by default until I configure JSON and after restarting the Mini it does not respond even after 10 or more minutes. I also failed to run the latest version of fullhaaboot after the flash.

**Comment by asbesto01** [comment]
I am having the same issue.

Fortunately I have downloaded a very old HAA version (2.2.1) and I have been able to install it offline (non OTA method).

@RavenSystem help needed to understand what's happening, seems HAA servers are down?

**Reply by Kienz** [reply]
Please post output from `nc -kulnw0 45678`

**Reply by msmicka** [reply]
Hi, now everything works for me without any problems. All the modules I was currently doing run on FW 7.4. Only the entire configuration needs to be done after logging in to the wifi HAA in the browser at 192.168.4.1:4567
In the HAA Manager mobile application, I will only check how the upgrade via mDNS takes place

After about 2 restarts, everything is set

**Comment by ErikDGC** [comment]
> 

I know this is an old post but Would you mind sharing the JSON configuration you used on your sonoff mini. As I've tried some I found online and homekit does detect it but the issue is the sonoff doesnt really reply to a fiscal switch it is connected to, but does thru homekit. Would really like if the fisical switch worked too, here is a video Of how I wish mine would work. https://m.youtube.com/watch?v=nrXdOczHWc4 

Please let me know, thanks in advance.
